components/Tabs/src/TabBarView/MDCTabBarSelectionIndicatorSupporting.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+/**
+ A simple protocol that indicates the responder can be targeted by an
+ @c MDCTabBarSelectionIndicatorTemplate.
+
+ @seealso MDCTabBarSelectionIndicatorTemplate
```

Comment:
Is this type in develop? I don't see it anywhere

Reply:
Yep, good catch.

Reply:
Done.